src/CollAction/Controllers/ProjectsController.cs

```diff
@@ -84,21 +103,21 @@ public ViewResult StartInfo()
         [HttpPost]
         [ValidateAntiForgeryToken]
         [Authorize]
-        public async Task<IActionResult> Create([Bind("Name,Description,Goal,CategoryId,LocationId,Target,End")] CreateProjectViewModel createProjectVM)
+        public async Task<IActionResult> Create([Bind("Name,Description,Goal,CategoryId,LocationId,Target,End")] EditProjectViewModel createProjectViewModel)
```

Comment:
EditProjectViewModel createProjectViewModel

Another issue with conflating the create and edit workflows